packages/malloy-db-duckdb/src/duckdb_wasm_connection.spec.ts

```diff
@@ -105,4 +105,11 @@ FROM read_parquet("inventory_items2.parquet")
       {}
     );
   });
+
+  // Test to check that DecimalBigNums are broken.
+  // Remove/update when fixed
```

Comment:
If it were me, I would mention `sqlLiteralNumber` in this comment, in case neither of us are here on the day this happens.